documentation/source/user_section/fsleyes.rst

```diff
@@ -11,6 +11,28 @@ be installed as either part of the ``FSL`` package, or as a standalone app.
 
 Previously, SCT provided instructions on how to install FSLeyes into the SCT environment. However, we now request that you install FSLeyes separately and manage the installation on your own. You can find installation instructions for FSLeyes at `this link <https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FSLeyes>`_.
 
+.. warning::
+
+   If you choose to install FSLeyes via a complete FSL installation, you will need to update 'wxpython' (an internal packaged used by FSLeyes). This is because the version of 'wxpython' that comes with FSL is out of date (`more info here <https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3988#issuecomment-1373918661>`_).
```

Comment:
Typo:
```suggestion
   If you choose to install FSLeyes via a complete FSL installation, you will need to update 'wxpython' (an internal package used by FSLeyes). This is because the version of 'wxpython' that comes with FSL is out of date (`more info here <https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3988#issuecomment-1373918661>`_).
```